exec/vm.go

```diff
@@ -60,6 +60,8 @@ type VM struct {
 	compiledFuncs []compiledFunction
 
 	funcTable [256]func()
+
+	recoverExecutionPanic bool
```

Comment:
Apologies but I am a bit confused now: I don't see how this could be set to `true` from outside the `exec` package.

Reply:
indeed, that's what you get for trying to optimize too early :flushed: . Should be fixed now.